features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/pom.xml

```diff
@@ -67,6 +88,46 @@
                                     <overWrite>true</overWrite>
                                     <outputDirectory>${basedir}/src/main/resources/runtimes/cxf3</outputDirectory>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.graalvm.sdk</groupId>
+                                    <artifactId>graal-sdk</artifactId>
+                                    <version>${graalvm.version}</version>
+                                    <type>jar</type>
+                                    <overWrite>true</overWrite>
+                                    <outputDirectory>${basedir}/src/main/resources/runtimes/lib</outputDirectory>
```

Comment:
let's change the outputDirectory
`${basedir}/src/main/resources/graal`

Change in other places a well.

Reply:
Modified